src/autoscale_daemon.py

```diff
@@ -67,7 +69,7 @@ def convert_to_sec(age):
     age -- age of node in the format of '4d', '6h', '23m', etc.
     """
 
-    age = re.findall('\d+[A-z]')[0]
+    age = re.findall('\d+[A-z]', age)[0]
```

Comment:
I think I solved this in another way in the current si330 deployment... because just finding the age was not enough... I have a local commit somewhere, I will push that soon alongside other things.